services/brig/src/Brig/API/Error.hs

```diff
@@ -37,16 +37,16 @@ import Network.HTTP.Types.Header
 import Network.HTTP.Types.Status
 import qualified Network.Wai.Utilities.Error as Wai
 import Servant.API.Status
-import Wire.API.ErrorDescription
+import qualified Wire.API.ErrorDescription as ErrDesc
```

Comment:
Why?

Reply:
I introduced a few `ErrorDescription`s with the same names as their counterparts in this module. Should I handle this in some other way?